lib/shopping_cart.rb

```diff
@@ -0,0 +1,64 @@
+class ShoppingCart
+  attr_reader :name,
+              :capacity,
+              :products,
+              :sorted,
+              :breakdown
+
+  def initialize(name, capacity)
+    @name = name
+    @capacity = capacity.to_i
+    @products = []
+    @sorted = []
+    @breakdown = []
```

Comment:
careful with assigning these two variables here. If you ever wanted to access `@sorted` or `@breakdown` in another method, it would be an empty array until you called the methods that filled them. For this reason, I think you should put these placeholders in the methods that fill them.